rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandlerSpec.groovy

```diff
@@ -354,6 +358,71 @@ class AWSBakeHandlerSpec extends Specification {
       1 * packerCommandFactoryMock.buildPackerCommand("", parameterMap, "$configDir/$awsBakeryDefaults.templateFile")
   }
 
+  void 'produces packer command with all required parameters for amzn, with sudo using runAsRoot:true'() {
+    setup:
+      def awsBakeryDefaultsJson = [
+        awsAccessKey: "FOO",
```

Comment:
I see the other tests below use these too. Perhaps they should be removed from those as well. It probably isnt wrong, but is misleading. I think. I assume these arent injected anywhere at runtime.